src/encoded/types/quality_metric.py

```diff
@@ -12,6 +12,47 @@
     lab_award_attribution_embed_list
 )
 
+QC_SUMMARY_SCHEMA = {
```

Comment:
A comment explaining what this is would be nice.

Reply:
Added.